core/commands/keystore.go

```diff
@@ -415,26 +426,28 @@ var keyRmCmd = &cmds.Command{
 
 func verifyIDFormatLabel(formatLabel string) error {
 	switch formatLabel {
-	case "b58mh":
-		return nil
-	case "b36cid":
+	case "b58mh", "v0":
 		return nil
+	default:
+		_, err := mbase.EncoderByName(formatLabel)
+		return err
 	}
-	return fmt.Errorf("invalid output format option")
 }
 
 func formatID(id peer.ID, formatLabel string) string {
 	switch formatLabel {
-	case "b58mh":
+	case "b58mh", "v0":
 		return id.Pretty()
```

Comment:
Let's use `id.IDB58Encode()` instead of `id.Pretty()` it's more precise in what we're looking for here and is never going to change. I know the function says it's deprecated, but it seems the safest to use.